### PR TITLE
Add MongoDB annotation generator

### PR DESCRIPTION
**Pull request by rabbitby**

### CODE REVIEW

File: src/AnnotationGenerator/DoctrineMongoDBAnnotationGenerator.php [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * (c) Andrew Meshchanchuk <andrew.meshchanchuk@gmail.com>
```
Comment (sroze) [inline src/AnnotationGenerator/DoctrineMongoDBAnnotationGenerator.php]: It is still the ApiPlatform project so you should use its license :)

File: src/AnnotationGenerator/DoctrineMongoDBAnnotationGenerator.php [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        $annotations = [];
+
+        if ($field['isEnum']) {
+            if ($field['isArray']) {
+                $type = 'simple_array';
+            } else {
+                $type = 'string';
+            }
+        } else {
+            switch ($field['range']) {
```
Comment (sroze) [inline src/AnnotationGenerator/DoctrineMongoDBAnnotationGenerator.php]: You can probably extract some parts of these method to other ones to make the code a bit more readable

**Comment by sroze** [comment]
Could you add tests please?

File: src/AnnotationGenerator/DoctrineMongoDBAnnotationGenerator.php (hunk 2 above)
Comment (rabbitby) [inline src/AnnotationGenerator/DoctrineMongoDBAnnotationGenerator.php]: I don't have idea, how do it. Can you help me?

**Comment by dunglas** [comment]
Thank you @rabbitby

**Pull request merged**